src/colorizer/utils/url_utils.ts

```diff
@@ -1,20 +1,26 @@
 // Typescript doesn't recognize RequestInit
 /* global RequestInit */
 
-import { DEFAULT_COLLECTION_FILENAME, DEFAULT_COLLECTION_PATH } from "../../constants";
+import { FeatureThreshold } from "../ColorizeCanvas";
+import { numberToStringDecimal } from "./math_utils";
 
 const URL_PARAM_TRACK = "track";
 const URL_PARAM_DATASET = "dataset";
 const URL_PARAM_FEATURE = "feature";
 const URL_PARAM_TIME = "t";
 const URL_PARAM_COLLECTION = "collection";
+// TODO: Make thresholds/filters language consistent. Requires talking with users/UX!
+const URL_PARAM_THRESHOLDS = "thresholds";
```

Comment:
Outstanding question of whether thresholds of filters is more familiar for our users. Will ask Lyndsay and possibly update this PR.

Reply:
since we use filter in the rest of the ui, it will be more consistent and understandable to use it in the url too.